src/components/AppContent/AppContent.vue

```diff
@@ -74,6 +190,30 @@ export default {
 				})
 			}
 		},
+		handlePaneResize(event) {
+			const paneSizes = {
+				list: event[0].size.toFixed(2),
+				details: event[1].size.toFixed(2),
+			}
+			browserStorage.setItem(this.paneIdentifier, JSON.stringify(paneSizes))
+		},
+		setContentHeight() {
+			this.contentHeight = window.innerHeight - document.getElementById('header').offsetHeight
```

Comment:
This currently breaks the docs on netlify:
```
TypeError: document.getElementById(...) is null
    setContentHeight webpack:///./src/components/AppContent/AppContent.vue?./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options:219
    mounted webpack:///./src/components/AppContent/AppContent.vue?./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options:186
    VueJS 14
    setCompiledPreview webpack:///./node_modules/vue-styleguidist/lib/client/rsg-components/Preview/PreviewAsync.js?:291
    componentDidMount webpack:///./node_modules/vue-styleguidist/lib/client/rsg-components/Preview/PreviewAsync.js?:138
    React 2
    unstable_runWithPriority webpack:///./node_modules/scheduler/cjs/scheduler.production.min.js?:19
    React 9
    render webpack:///./node_modules/vue-styleguidist/lib/client/index.js?:46
    <anonymous> webpack:///./node_modules/vue-styleguidist/lib/client/index.js?:54
    js https://deploy-preview-1759--nextcloud-vue-components.netlify.app/build/bundle.1310bebb.js:13447
    __webpack_require__ https://deploy-preview-1759--nextcloud-vue-components.netlify.app/build/bundle.1310bebb.js:64
    <anonymous> webpack:///multi_./styleguide/global.requires.js_./styleguide/assets/icons.css_./styleguide/assets/additional.scss_./node_modules/vue-styleguidist/lib/client/index?:4
    0 https://deploy-preview-1759--nextcloud-vue-components.netlify.app/build/bundle.1310bebb.js:19711
    __webpack_require__ https://deploy-preview-1759--nextcloud-vue-components.netlify.app/build/bundle.1310bebb.js:64
    <anonymous> https://deploy-preview-1759--nextcloud-vue-components.netlify.app/build/bundle.1310bebb.js:199
    <anonymous> https://deploy-preview-1759--nextcloud-vue-components.netlify.app/build/bundle.1310bebb.js:202
vue.esm.js:1743:13
```

Reply:
Hi and thanks for the feedback! How would I be able to reproduce this? 

Fixing this should be simple but I don't quite understand why this is happening in the first place, as this would only happen in case of a missing header (which seems unlikely). Any insight into why the build process is doing things differently than the regular build process?

Reply:
Well, like I said - I have no way of reproducing this so I am flying blind, but I made a change that makes the offending line a little more verbose but should get rid of the error you're seeing. Otherwise this PR is gonna be still open next christmas :P